### ISSUE TITLE
Binary Search - Leet Code

### ISSUE THREAD

**Issue by Chirag-007** [kanak22/CP-DSA-Questions#575]
## Link of the question
Provide the link of the question and ask for assignment of the issue.
https://leetcode.com/problems/binary-search/
## Coding platform

- [ ] Hackerrank
- [ ] HackerEarth
- [ ] Codechef
- [ ] Codeforces
- [x] Leetcode
- [ ] InterviewBit
- [ ] AtCoder
- [ ] 450 DSA Sheet
## Programming language

- [ ] C
- [ ] C++
- [ ] Java
- [ ] Python

**Comment by Chirag-007** [comment]
Hey @kanak22 ! Please assign this issue to me in Python (Fork-N-Commit)

**Comment by Sxmxrth** [comment]
Hey @kanak22 !! Please assign this issue to me in Java( Fork-N-Commit)

**Comment by kanak22** [comment]
Okay @Chirag-007 you do this in python and @Sxmxrth you can do this in java

**Comment by kanak22** [comment]
@Chirag-007 You should never merge your own PR and u should never self assign it as it can create problems for the project admin. Remember this always. 😊 for now i will seeing it nd if doesn't goes with the guidelines then i will be reverting it.

**Comment by Chirag-007** [comment]
Yeah ma'am, i'm trying to revert it but can't seem to, can you revert it back from you end if possible?

**Comment by kanak22** [comment]
Yes i reverted it from the main project 😊

**Comment by Chirag-007** [comment]
Ok ma'am, thank you and sorry for doing that.

**Comment by kanak22** [comment]
Its okay

**Comment by Navaneethp007** [comment]
Hey @kanak22, please assign me this issue in C++ (Fork N Commit FnC15)
